orion/primitives/timeseries_anomalies.py

```diff
@@ -1,10 +1,13 @@
 """
 Time Series anomaly detection functions.
-Implementation inspired by the paper https://arxiv.org/pdf/1802.04431.pdf
+Some of the implementation is inspired by the paper https://arxiv.org/pdf/1802.04431.pdf
 """
+import math
 
 import numpy as np
 import pandas as pd
+import similaritymeasures as sm
```

Comment:
I remember you know a dtw library which is more efficient in speed? Can we use that one here?

Reply:
Also, add a constraint to dtw would make more sense to me in our scenario

Reply:
There are two packages worth considering:
* https://github.com/pierre-rouanet/dtw
* https://github.com/johannfaouzi/pyts

I will make another comparison test and make an update

Reply:
Yes, `pyts` is the best one!